dali/test/python/test_video_pipeline.py

```diff
@@ -34,11 +34,18 @@
 BATCH_SIZE=4
 COUNT=5
 
+def get_gpu_num():
```

Comment:
maybe move this to `test_utils.py`

Reply:
test_utils.py imports `PIL` and impose dependency on that, and whoever is uses that needs to have `PIL` installed. I can rework test_utils.py to import this package only when given functionality is used.

Reply:
Done